stapi_fastapi/main_router.py

```diff
@@ -81,6 +80,7 @@ def __init__(
             tags=["Orders"],
             response_model=Order,
         )
```

Comment:
This route can be removed as its functionality will be handled by the product router.

Reply:
You will also need to update the `products()` implementation to walk the products that have been added to this router and build the list response of those products. And the `create_order()` and `product()` methods can be removed as they will be handled by the product router(s).

Reply:
worked through this in person